src/components/castleRisk/initialCastleRiskState.ts

```diff
@@ -0,0 +1,5 @@
+import { CastleRiskInitialState } from "./game";
```

Comment:
here and lots of these "initial...State.ts" files just define the initial state, but closer to the folder they belong to. state gets imported and cobbled together in the same place as before, just pulling from these several more specific locations...